app/helpers/application_helper.rb

```diff
@@ -1,2 +1,26 @@
 module ApplicationHelper
+  class << self
+    def reduce_same_keys(arr)
```

Comment:
Name of a function should probably not use a generic term like `reduce`, `loop`, etc. but rather a description of what it does more practically.